### PR TITLE
Review my project module one pull request

### PR DESCRIPTION
**Pull request by AleWaweru**
Created a simple html file
created css file
created javascript file

### CODE REVIEW

**Comment by AleWaweru** [comment]
good job here @AleWaweru

great design
good github flow
well descriptive pull request title
well descriptive pull request details
great use of HTML/CSS best practises
you are almost there, but some changes are required for you proceed to the next project.

Kindly install linters, this helps you manage errors in your css files.
Your ReadMe file should include all that you did in your project.This helps another developer understand how the project was build and the tools used.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @AleWaweru , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
✅Added descriptive comments to review.


 ## Required Changes ♻️

**1-** Please consider not using the words like `should` etc which makes the user feels that you're ordering him/her. Always be polite and consider using the words and phrases like `Please consider, kindly, etc` to make it user-friendly behavior.

**2-** Please always be more descriptive about what you are asking. Regarding your README comment, there is a small confusion that what actually the user needs to add in the README like any specific section, code, links, or what? 

### DEMO:-

Following is a more clear, descriptive, polite comment also having a reason:--

```
The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?
```

**3-** Please consider using the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments as its a project requirement.

* Make sure to leave the comment under a line or block of code that fits the context the comment is about.


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (AleWaweru) [inline README.md:1]: The ReadMe's  "Built With" and "Getting Started " sections are currently empty.I kindy suggest you add content, listing the technology used in the project and how to run it.This makes your project look more proffesional.

File: html/index.html
Comment (AleWaweru) [inline html/index.html:1]: fantistic use of html best practises

**Review by AleWaweru — COMMENTED** [review]
great design
good use of gitflow
well descriptive pull request title
well descriptive pull request details
great use of HTML/CSS best practises
you are almost there, but some changes are required for you proceed to the next project.

Kindly install linters, this helps you manage errors in your css files.

please consider the comments made and make the changes as required.

Cheers.

File: README.md
Comment (ABDULALI3468) [reply README.md:1]: > The ReadMe's "Built With" and "Getting Started " sections are currently empty.I kindy suggest you add content, listing the technology used in the project and how to run it.

Good job but please describe `why` the changes are necessary like `to make the project professional or it a project requirement ` to make the user clear why the changes are being asked. Moreover, 


[Make sure to leave the comment under a line or block of code that fits the context the comment is about.](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @AleWaweru , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅Used GitHub inline comments.
✅Added descriptive comments.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AleWaweru — COMMENTED** [review]
changes made,great work

**Review by AleWaweru — COMMENTED** [review]
changes made, great work.

File: README.md
Comment (AleWaweru) [inline README.md:1]: The ReadMe's "Built With" and "Getting Started " sections are currently empty.I kindy suggest you add content, listing the technology used in the project and how to run it.This makes your project look more proffesional.

**Review by AleWaweru — COMMENTED** [review]
changes made.Great work.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @AleWaweru 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Good explanation of why to perform the changes. ✔️ 
 - Good use of inline comments. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**